scripts/build/wheel_macos_before_all.sh

```diff
@@ -11,6 +11,12 @@ set -x
 dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 prjdir="$( cd "${dir}/../.." && pwd )"
 
+# Build dependency libraries
+"${prjdir}/scripts/build/build_libpq.sh"
+
+# Show dependency tree
+# otool -L /tmp/libpq.build/lib/*.dylib
```

Comment:
Did you mean to comment the `otool` line out?

Reply:
Yes, I thought it was just a debug statement. Do we need its output in CI?

Reply:
It is just debug but it's quite nice to see exactly why some homebrew package got sucked in every time cibuildwheel gives you that rather unhelpful error about raising deployment targets. It's also handy for answering your `gettext` question.

It's up to you.

Reply:
Fair. Let me enable it back :slightly_smiling_face: